lib/membrane_opus.ex

```diff
@@ -8,31 +8,31 @@ defmodule Membrane.Opus.Description do
   @typedoc """
   Number of channels transmitted in the stream.
   """
-  @type channels_t :: 1 | 2
+  @type channels_t :: 1 | 2 | nil
```

Comment:
So far we didn't allow that (`nil`s in caps). Do you want to change that? `Membrane.Opus` struct with nils would mark unparsed stream? Still it seems not usable as File.Source would not provide such format in caps

Reply:
File.Source would not, but RTP.Depayloader does. At the same time, it outputs the stream chunked by Opus packets.

Reply:
Maybe the Depayloader should handle that? Getting channels and bitrate and channels is just a matter of interpreting first byte of this packet

Reply:
I believe it should be done by the parser, but the parser isn't always needed